{"name": "test_upload", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'E:\\\\图片\\\\Saved Pictures\\\\鬼灭之刃1.jpg'", "trace": "self = <request.case.test_01.TestRequest object at 0x000001CDAF3809D0>\n\n    @allure.story(\"上传文件\")\n    def test_upload(self):\n        '''图片上传'''\n        url='http://116.63.186.61:2031/api/Common/Files/Upload'\n        headers={'Authorization': self.Authorization}\n    \n        file={\n            'module': 'Goods',\n>           'Files': ('弥豆子.jpg', open('E:\\图片\\Saved Pictures\\鬼灭之刃1.jpg','rb'), 'image.png')\n        }\nE       FileNotFoundError: [Errno 2] No such file or directory: 'E:\\\\图片\\\\Saved Pictures\\\\鬼灭之刃1.jpg'\n\ntest_01.py:44: FileNotFoundError"}, "description": "图片上传", "start": 1667219397535, "stop": 1667219397535, "uuid": "113bdeb5-244d-4cb9-b298-e0ed6e7ebba0", "historyId": "d980c573a84ef58b04388da3b8c86470", "testCaseId": "cff5be8551d4032f38e0157b6f7135df", "fullName": "test_01.TestRequest#test_upload", "labels": [{"name": "feature", "value": "测试类名"}, {"name": "story", "value": "上传文件"}, {"name": "suite", "value": "test_01"}, {"name": "subSuite", "value": "TestRequest"}, {"name": "host", "value": "Liukang-PC"}, {"name": "thread", "value": "7756-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_01"}]}